packages/twenty-front/src/modules/object-metadata/hooks/useFieldMetadataItem.ts

```diff
@@ -51,24 +51,24 @@ export const useFieldMetadataItem = () => {
       | 'options'
     >,
   ) => {
-    const formattedInput = formatFieldMetadataItemInput(input);
-    const defaultValue = input.defaultValue
-      ? typeof input.defaultValue == 'string'
-        ? `'${input.defaultValue}'`
```

Comment:
we should not do this quote prefixing and suffixing here in the generic updateField function. This should be done while writing in the defaultValue. Otherwise this will lead to re-added quotes